junit5/src/main/java/cz/xtf/junit5/annotations/SkipFor.java

```diff
@@ -16,9 +16,20 @@
 public @interface SkipFor {
 
 	/**
-	 * Name or regexp pattern matching name of the image. For example "eap73-openjdk11-openshift-rhel8" or "eap73-openjdk11-.*"
+	 * Name or regexp pattern matching name of the image.
+	 * For example "eap73-openjdk11-openshift-rhel8" or "eap73-openjdk11-.*".
+	 *
+	 * One of {@code dockerLabelName} or {@code name} must be present.
```

Comment:
I think here should be provided name of the property from which the name is taken so it's clear what to specify here.

Reply:
Please check.